### ISSUE TITLE
Version tagging, release

### ISSUE THREAD

**Issue by danmichaelo** [NatLibFi/Skosify#43]
Could you tag the 1.0.0 version using  `git tag` and publish a [Github release](https://github.com/NatLibFi/Skosify/releases)? Perhaps also a version 1.0.1 that incorporates the latest changes?

Would also be great if you could submit the package to PyPI. It's [quite easy](http://peterdowns.com/posts/first-time-with-pypi.html)

**Comment by osma** [comment]
Good ideas, thanks.

I've created a version tag (v1.0) and GitHub [release](https://github.com/NatLibFi/Skosify/releases/tag/v1.0) for the original 1.0 release.

Publishing a new version with all the latest changes would make sense too, though I'm unsure about the version number. [Semantic versioning](http://semver.org/) principles tell that API changes should warrant new major versions. The work you did to separate Skosify into its own package may or may not be significant in this respect. At least it changed the directory structure, so if old scripts used the old location of skosify.py, they need to be changed.

PyPI publication makes sense too, but I created a separate issue for that. Doing that should wait for the new release IMHO. I cannot see an urgent need to publish 1.0 on PyPI as it won't work as a Python package anyway.
